modules/homa-lite/src/lib.rs

```diff
@@ -176,13 +176,13 @@ pub mod module {
 				.checked_mul_int(
 					amount
 						.checked_sub(T::MintFee::get())
-						.expect("Arithmatic should not panic"),
+						.expect("Mint amount is ensured to be greater than T::MintFee; qed"),
 				)
 				.ok_or(ArithmeticError::Overflow)?;
 
 			liquid_to_mint = liquid_to_mint
 				.checked_sub(T::MaxRewardPerEra::get().mul(liquid_to_mint))
```

Comment:
Not introduced in this PR but would be good to change it to `checked_mul`

Reply:
I think checked_mul_int is correct here - it takes in an u128 and produces an u128. CheckedMul gives FixedU128 and I had to convert it back to u128.

Reply:
Yes, should use `checked_mul_int`.

Reply:
Sorry my mistake - i was looking at wrong line of code. For T::MaxRewardPerEra::get().mul(liquid_to_mint), checked_mul is correct. Mainly because checked_mul_int is implemented only for FixedU128, and not for U128.

Reply:
Update: Refer to https://github.com/AcalaNetwork/Acala/issues/1321